modules/programs/saml2aws.nix

```diff
@@ -0,0 +1,94 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.saml2aws;
+  iniFormat = pkgs.formats.ini { };
+  inherit (lib) mkIf mkOption types;
+in
+{
+  meta.maintainers = [ lib.hm.maintainers.mokrinsky ];
+
+  options.programs.saml2aws = {
+    enable = lib.mkEnableOption "saml2aws CLI tool";
+
+    package = lib.mkPackageOption pkgs "saml2aws" {
+      default = "saml2aws";
+      nullable = true;
+    };
+
+    enableBashIntegration = lib.hm.shell.mkBashIntegrationOption {
+      inherit config;
+      extraDescription = ''If enabled, this will install autocompletion for bash.'';
```

Comment:
If the completions are the only benefit of the integration, aren't the shell completions already installed https://github.com/NixOS/nixpkgs/blob/c4e9b72d0bf394509a3b2aff27be35b7e3c99b85/pkgs/by-name/sa/saml2aws/package.nix#L33-L37